src/client/eppo-client.ts

```diff
@@ -206,49 +254,91 @@ export default class EppoClient {
    * @param subjectKey an identifier of the experiment subject, for example a user ID.
    * @param subjectAttributes optional attributes associated with the subject, for example name and email.
    * @param defaultValue default value to return if the subject is not part of the experiment sample
-   * @returns a number variation value if the subject is part of the experiment sample, otherwise the default value
+   * @returns a boolean variation value if the subject is part of the experiment sample, otherwise the default value
```

Comment:
"integer"?

Reply:
yeah I noticed that just a few minutes ago. I'll update

Reply:
👆

Reply:
@aarsilv This was fixed in https://github.com/Eppo-exp/js-client-sdk-common/pull/100